src/scatter_buildblock/ScatterSimulation.cxx

```diff
@@ -959,8 +959,7 @@ ScatterSimulation::downsample_scanner(int new_num_rings, int new_num_dets)
     
     const float approx_num_non_arccorrected_bins =
       this->proj_data_info_sptr->get_num_tangential_poss() * 
-      (float(new_num_dets) / old_scanner_ptr->get_num_detectors_per_ring())
-      + 5; // add 5 to avoid strange edge-effects, certainly with B-splines
```

Comment:
@KrisThielemans: I removed this line, because it invalidates the checks for #views and #tangential bins being the same for downsampled BOC scanners. I am already extending the sinograms for Cylindrical scanners in tangential direction at the interpolation stage (where it wasn't done in the past), so this should no longer be required here. Do you agree?